pspamm/scripts/max_knl.py

```diff
@@ -16,4 +16,4 @@ def getBlocksize(m , n, bk):
 
 
 def KNL_condition(bm, bn, bk):
-    return (bn+bk) * (bm / 8) + 2 <= 32
+    return (bn+bk) * (bm / 8) <= 32
```

Comment:
ceiling division?

Reply:
Similar to `scripts/max_arm.py` where `bm` a multiple of `v_size` (in this case a constant `8`).